package.json

```diff
@@ -98,7 +98,6 @@
     "redux-localstorage": "^0.4.1",
     "resin-cli-form": "^1.4.1",
     "resin-cli-visuals": "^1.2.8",
-    "rindle": "^1.3.0",
```

Comment:
Haha, that @jviotti writes such [wasteful code](https://www.npmjs.com/package/rindle) :rofl: